### PR TITLE
Add Tooling

### PR DESCRIPTION
**Pull request by rijkvanzanten**

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -1,10 +1,7 @@
 // Require all the packages
 const http = require('http');
+const path = require('path');
 const express = require('express');
-const OBA = require('oba-api');
-const debug = require('debug')('robat');
```
Comment (dandevri) [inline server/index.js:5]: Why remove these `requires`?

File: package.json [hunk 2]
```diff
@@ -27,7 +37,19 @@
     "uuid": "^3.0.1"
   },
   "devDependencies": {
```
Comment (dandevri) [inline package.json:39]: I know you like tooling @rijkvanzanten but isn't all of this a bit too much? Linters seem okay but all the babel and browserify seems a bit over the top?

Comment (dandevri) [reply package.json:39]: Not exactly sure why it's needed and overcomplicate things? Correct me if I'm wrong. 🐸

File: .stylelintrc [hunk 3]
```diff
@@ -0,0 +1,129 @@
+{
```
Comment (dandevri) [inline .stylelintrc:1]: Are these all the default stylelint rules or did you add / removed some?

**Review by dandevri — CHANGES_REQUESTED** [review]
🌮

File: package.json (hunk 2 above)
Comment (rijkvanzanten) [reply package.json:39]: We were planning on using stuff like localForage & axios for the client, which needs a bundler. Babelify is to transpile the modules to es5 to allow a greater browser support with less effort from our side.

If you'd like to see some over the top sh*t, please checkout [@Mimaaa's Webpack setup](https://github.com/rijkvanzanten/obachine/blob/master/webpack.parts.js)

File: .stylelintrc (hunk 3 above)
Comment (rijkvanzanten) [reply .stylelintrc:1]: These are the same rules used in [our other project, Obachine.](https://github.com/rijkvanzanten/obachine/blob/master/.stylelintrc)

File: server/index.js (hunk 1 above)
Comment (rijkvanzanten) [reply server/index.js:5]: To fix the linting errors ^^

Also, the OBA api handler is probably going to be used in a separate file and there wasn't a single debug statement necessary in the main index file at this point

File: package.json (hunk 2 above)
Comment (dandevri) [reply package.json:39]: 🔥 Seems legit

File: server/index.js (hunk 1 above)
Comment (dandevri) [reply server/index.js:5]: Alright cool!

File: .stylelintrc (hunk 3 above)
Comment (dandevri) [reply .stylelintrc:1]: Good way to keep both of the projects consistent 🎉

**Review by dandevri — APPROVED** [review]

**Pull request merged**